app/views/help_requests/index.html.haml

```diff
@@ -0,0 +1,5 @@
+%h3 Help request messages
+
+- @help_requests.last(20).each do |help|
```

Comment:
If you want to limit the query to the latest 20 records you could (and imo should) do that in your controller:
```ruby
HelpRequest.order(id: :desc).limit(20)
```